pulumi/src/backupCron.ts

```diff
@@ -1,92 +1,110 @@
-import { StatefulService } from ".";
 import * as k8s from '@pulumi/kubernetes'
+import { StatefulService } from '.'
 
-export const deployStsBackupCron = (asset: string, sts: StatefulService, namespace: string, provider: k8s.Provider) => {    
+export const deployStsBackupCron = (asset: string, sts: StatefulService, namespace: string, provider: k8s.Provider) => {
   if (!sts.backup) return
 
-  const serviceAccountName = createRbac(asset, namespace, provider)
-  const svcs = sts.services.map(svc => svc.name).join(',')
+  const name = `${asset}-volume-reaper`
+  const serviceAccountName = createRbac(name, namespace, provider)
+  const services = sts.services.map((svc) => svc.name).join(',')
 
-  const backupContainer = {
-    name: `${asset}-backup-runner`,
-    image: 'shapeshiftdao/volumereaper:latest',
-    imagePullPolicy: "Always",
```

Comment:
use inferred `imagePullPolicy`. `:latest` defaults to `Always`, `:someNonLatestTag` defaults to `IfNotPresent`